target/linux/octeon/image/Makefile

```diff
@@ -29,6 +29,16 @@ define Device/generic
 endef
 TARGET_DEVICES += generic
 
+ITUSROUTER_CMDLINE:=console=ttyS0,115200, root=/dev/mmcblk1p2 rootfstype=squashfs,ext4,f2fs rootwait
+define Device/itus,shield-router
+  DEVICE_VENDOR := Itus Networks
+  DEVICE_MODEL := Shield Router
+  CMDLINE := $(ITUSROUTER_CMDLINE)
+  IMAGE/sysupgrade.tar/squashfs :+ append-metadata | combined-image
+  CPU_TYPE := octeon3
```

Comment:
CPU_TYPE is a _target_ variable. I suspect it won't do anything if defined here?

Reply:
I suspect as much as well, which is why I was asking about whether I should be doing this as a subtarget.. octeonplus is a very OLD platform of octeon, and the kernel checks and accommodates octeon3 vs octeonplus differently.